pym/portage/dbapi/bintree.py

```diff
@@ -672,17 +693,17 @@ def _populate(self, getbinpkgs=False, getbinpkg_refresh=True):
 					multi_instance = False
 					invalid_name = False
 					build_id = None
-					if myfile.endswith(".xpak"):
+					if myfile.endswith("." + self.suffix):
 						multi_instance = True
 						build_id = self._parse_build_id(myfile)
```

Comment:
_parse_build_id also seems to return strings instead of a long sometime because the method catches the ValueError. @zmedico Any advice here?